app/services/movie_service.rb

```diff
@@ -5,56 +5,45 @@ def get_data(url)
     JSON.parse(data, symbolize_names: true)
   end
 
-  def movies_by_vote_average
-    url = 'https://api.themoviedb.org/3/discover/movie?api_key=cc1b7a1d937de5062ee5336bdb03e44d&language=en-US&sort_by=vote_average.desc&include_adult=false&include_video=false&page=1&vote_count.gte=100&with_original_language=en'
-    url_2 = 'https://api.themoviedb.org/3/discover/movie?api_key=cc1b7a1d937de5062ee5336bdb03e44d&language=en-US&sort_by=vote_average.desc&include_adult=false&include_video=false&page=2&vote_count.gte=100&with_original_language=en'
-    movie_data = get_data(url)
-    movie_data_2 = get_data(url_2)
-    total_results = movie_data[:results].concat(movie_data_2[:results])
+  def top_forty_movies
     result = {}
-    total_results.each do |movie|
-      result[movie[:title]] = movie[:id]
-      result[movie[:title]] = movie[:vote_average]
+    2.times do |n|
+      movie_data = get_data(url_storage(num: n)[:movie_top_forty])
+      movie_data[:results].each do |movie|
+        result[movie[:title]] = movie_id_and_vote_average(movie)
+      end
```

Comment:
Neat! This is basically combining what we had originally worked out with a times loop and what you were working on above. I like how much more concise this is and that there's less clutter from the whole url inputs